<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialog</class>
 <widget class="QDialog" name="InputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="b_0">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ok">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_number">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
